test/framework/cluster_proxy.go

```diff
@@ -103,6 +107,21 @@ type ClusterProxy interface {
 	Dispose(context.Context)
 }
 
+// CreateOrUpdateConfig contains options for use with CreateOrUpdate.
+type CreateOrUpdateConfig struct {
+	LabelSelector string
+}
```

Comment:
```suggestion
// createOrUpdateConfig contains options for use with CreateOrUpdate.
type createOrUpdateConfig struct {
	labelSelector string
}
```

Let's keep this one private.